sdk/eventhub/Azure.Messaging.EventHubs.Shared/src/Testing/LiveResourceManager.cs

```diff
@@ -30,241 +30,281 @@ public sealed class LiveResourceManager
+            /// </summary>
+            ///
+            /// <param name="message">The <see cref="HttpMessage" /> for the operation.</param>
+            /// <param name="exception">The exception, if any, thrown by the operation.</param>
+            ///
+            /// <returns><c>true</c> if the operation should be retried; otherwise, <c>false</c>.</returns>
+            ///
+            protected override bool ShouldRetry(HttpMessage message, Exception exception)
+            {
+                var originalClassifier = message.ResponseClassifier;
 
-            /// <summary>The date and time, in UTC, that the token expires.</summary>
-            public readonly DateTimeOffset ExpiresOn;
+                try
+                {
+                    message.ResponseClassifier = Classifier;
+                    return base.ShouldRetry(message, exception);
+                }
+                finally
+                {
+                    message.ResponseClassifier = originalClassifier;
+                }
+            }
 
             /// <summary>
-            ///   Initializes a new instance of the <see cref="ManagementToken"/> class.
+            ///   Determines whether a retry should be performed for the associated operation.
             /// </summary>
             ///
-            /// <param name="token">The value of the bearer token.</param>
-            /// <param name="expiresOn">The date and time, in UTC, that the token expires on.</param>
+            /// <param name="message">The <see cref="HttpMessage" /> for the operation.</param>
+            /// <param name="exception">The exception, if any, thrown by the operation.</param>
+            ///
+            /// <returns><c>true</c> if the operation should be retried; otherwise, <c>false</c>.</returns>
             ///
-            public ManagementToken(string token,
-                                   DateTimeOffset expiresOn)
+            protected override ValueTask<bool> ShouldRetryAsync(HttpMessage message, Exception exception)
             {
-                Token = token;
-                ExpiresOn = expiresOn;
+                var originalClassifier = message.ResponseClassifier;
+
+                try
+                {
+                    message.ResponseClassifier = Classifier;
+                    return base.ShouldRetryAsync(message, exception);
+                }
+                finally
+                {
+                    message.ResponseClassifier = originalClassifier;
+                }
             }
+
+            /// <summary>
+            ///   A custom response classifier that extends the common retry scenarios to include
+            ///   additional response types specific to Event Hubs management operations.
+            /// </summary>
+            ///
+            /// <seealso cref="Azure.Core.ResponseClassifier" />
+            ///
+            private class EventHubsManagementClassifier : ResponseClassifier
```

Comment:
The conditions for retry here are more extensive and looser than the default retry policy.  These were curated from pipeline failures over the last 3 years and I don't think they're generally desirable outside this scenario - even if just scoped to the management packages.